### DISCUSSION TITLE
SP2013 to Online and SharePoint Rest Service exception

### DISCUSSION THREAD

**Discussion by ririikon** [pnp/modernization#599] [Q&A]
I'm having few thousand pages to modernize and 2/3 goes well. Still one third is getting this error "Page not transformed" and reason is that "SharePoint Rest Service exception" at PnP.Core.Services.BatchClient.ProcessSharePointRestBatchResponseContent (Batch batch, String batchResponse), but it doesn't open situation for me at all. Any hints how to make this work?

Could it be something to do with renaming the page files? I'm removing special characters from variable $targetFileName and ConvertTo-PnPPage uses that new name as PublishingTargetPageName. With my logic it shouldn't affect.

**Comment by jansenbe** [comment]
@ririikon : please create an issue in https://github.com/pnp/pnpframework/issues, page transformation tech is now a part from PnP Framework. For the failing page enable verbose logging and the log output (after scrubbing) to the issue.
